### PR TITLE
10 app navigation

### LINKED ISSUE #10: Create app Navigation
Create app Navigation for our react native app

### PR DESCRIPTION
**Pull request by reemsh2000**
* create app navigation

### CODE REVIEW

File: app/screens/Chat.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <SafeScreen style={styles.container}>
+      <AppText styles={styles.text}> Hi Chat </AppText>
+    </SafeScreen>
+  );
+}
+
+const styles = StyleSheet.create({
+  container:{
+    backgroundColor:'#000',
```
Comment (Nizar7zak) [inline app/screens/Chat.js:18]: I think you forget use colors file
colors.black
colors.white

**Review by Nizar7zak — APPROVED** [review]
Great Work !

File: app/screens/Welcome.js [hunk 2]
```diff
@@ -85,6 +83,7 @@ const styles = StyleSheet.create({
 
   loginButton: {
     backgroundColor: colors.primary,
+    marginBottom:50
```
Comment (mazenchami) [inline app/screens/Welcome.js]:
```suggestion
    marginBottom: 50,
```

**Review by mazenchami — APPROVED** [review]
Looks great!

**Pull request merged**